src/rules/indentation.js

```diff
@@ -16,41 +16,46 @@ var availableConfigs = {
 
 var errors = [];
 
-function test(parsedLocation, config, type) {
+function test(parsedLocation, configuration, type) {
     // location.column is 1 index based so, when we compare with the expected indentation we need to subtract 1
-  if (--parsedLocation.column !== config[type]) {
-    errors.push({message: 'Wrong indentation for "' + type + '", expected indentation level of ' + config[type] + ', but got ' + parsedLocation.column,
+  if (--parsedLocation.column !== configuration[type]) {
+    errors.push({message: 'Wrong indentation for "' + type + '", expected indentation level of ' + configuration[type] + ', but got ' + parsedLocation.column,
                  rule   : rule,
                  line   : parsedLocation.line});
   }
 }
 
+function testStep(step, language, configuration, mergedConfiguration) {
+  var keyword = step.keyword;
+  var stepType = _.findKey(language, function(values) { return values instanceof Array && values.indexOf(keyword) !== -1; });
+  stepType = stepType in configuration ? stepType : 'Step';
```

Comment:
This is what actually does the fallback: 
- If I have specified the stepType in the config file, use my configuration, otherwise use the 'Step' configuration.